data/scenarios/Challenges/arbitrage.yaml

```diff
@@ -0,0 +1,413 @@
+version: 1
+name: Paperclip maximizer
+author: Karl Ostmo
+description: |
+  Make trades to earn your fortune.
+creative: false
+objectives:
+  - teaser: Earn riches
+    goal:
+      - |
+        Each shop offers different wares for trade.
+        Recipes dictate which items may be exchanged
+        at any given shop. Use the `drill` on a shop to perform
+        an exchange.
+      - |
+        As an itinerant merchant, you may exploit market asymmetry
+        for profit.
+      - |
+        Amass a fortune of 100 paperclips.
+    condition: |
+      as base {
+        pcount <- count "paperclip";
+        return $ pcount >= 100;
+      };
+  - teaser: Beginner capital
+    goal:
+      - |
+        Great start! Perhaps it is time
+        to enter a higher-end goods market?
+    optional: true
+    hidden: true
+    condition: |
+      as base {
+        pcount <- count "paperclip";
+        return $ pcount >= 20;
+      };
+attrs:
+  - name: shopA
+    fg: '#ff0000'
+  - name: shopB
+    fg: '#00ff00'
+  - name: shopC
+    fg: '#0000ff'
+  - name: shopD
+    fg: '#ffff00'
+  - name: shopE
+    fg: '#ff00ff'
+  - name: shopF
+    fg: '#00ffff'
+  - name: shopG
+    fg: '#bb8844'
+  - name: shopX
+    fg: '#000000'
+    bg: '#ffffff'
+  - name: shopY
+    fg: '#ffffff'
+    bg: '#000000'
+robots:
+  - name: base
+    dir: [1, 0]
+    devices:
+      - ADT calculator
```

Comment:
How about equipping a `compass`?  Just for convenience when programming the base and/or exploring initially.